private-templates-service/client/src/components/NavBar/styled.tsx

```diff
@@ -1,9 +1,47 @@
 import styled from 'styled-components';
+import {BREAK} from '../../globalStyles';
 
-export const AvatarIcon = styled.img`
-  width: 32px;
+export const Banner = styled.div`
+  background-color: #6F6F6F;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  padding: 8px 64px;
+  z-index: 1;
+
+  @media only screen and (max-width: ${BREAK.SM}px){
+    padding: 8px 24px;
+  }
+`;
+
+export const MobileBanner = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
 `;
 
-export const DefaultAvatarIcon = styled.i`
-  width: 32px;
+export const Logo = styled.img`
+  display: none;
+
+  @media only screen and (max-width: ${BREAK.SM}px){
```

Comment:
rather than having `display: none` as default styles and `display: block` as mobile style, make the default style `display: block` and have `min-width: BREAK.SM` cause `display: none`